.github/workflows/build.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build and Push Consumer Image
+on:
+  push:
+    branches: [ main, release-4.18 ]
```

Comment:
Is it ok to run on all branches? This way we don't need to update this later:
```
branches:
  - "*"
```

Reply:
we do not want to run on all branches. 4.18 is the last version to support v1 events. For v2 events we always use main branch.